.github/workflows/build-windows.yml

```diff
@@ -141,6 +127,15 @@ jobs:
           submodules: true
           fetch-depth: 0
 
+      - name: Setup MSVC toolset
+        uses: TheMrMilchmann/setup-msvc-dev@v3
+        with:
+          arch: x64
+          # Use the latest toolset available
+          toolset: 14.XX
```

Comment:
Without this, CMake uses MinGW. 😅 But I made it future-proof by not pinning to a specific toolset version.